From 4c98cdc6c0222c0ef1658f7b47b2bce6757620a6 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 18 Feb 2016 16:45:13 +0000
Subject: [PATCH 0733/1076] MANTIS 0037187 extension/bugfix: CUSS-Spec:
 DS_TYPES_SSML11= 25010

codes.idl
V1.3.5


added DS_TYPES_SSML10; DS_TYPES_SSML11
- clarified: use DS_TYPES_SSML11= 25010 instead of DS_TYPES_SSML11= 25001



git-svn-id: svn://localhost/SelfServiceCommon/trunk@989 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 98 +++++++++++----------
 1 file changed, 53 insertions(+), 45 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index c495b2a5..e992b5a1 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -1,48 +1,53 @@
-//---------------------------------------------------------------------------
-//
-//  File:      codes.idl
-//
-//  Purpose:   Definition of CUSS codes
-//
-//  Date:      17.06.2013
-//
-//  Version:   1.3
-//
-//  Author:    IATA Passenger Experience Management Group: CUWG CUSS-TSG
-//
-//  Copyright(c) 2003,2009,2013 International Air Transport Association, All Rights Reserved
-//
-//  Note:      Please refer to the CUSS 1.3 Technical Specification for more information
-//
-//  Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
-//
-//  @date 24.01.2013
-//  @li V1.3
-//  @li Mantis 0025566: WeightScale: support standalone Scale (BaggageScale CUSS1.3)
-//  @li added CUSS1.3 dataStatus and statusCodes
-//
-//  @date 26.11.2013
-//  @li V1.3.1
-//  @li Mantis 0029957: ConveyorComp: CUSS1.3 Conveyor (without ConveyorAEA)
-//  @li use full CUSS1.3 IDL + Materna extension statuscodes: BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, 
-//
-//  @date 26.01.2016
-//  @li V1.3.2
-//  @li Mantis 0037350: integration of EZ-Access device for KIS kiosk
-//  @li added DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
-//  @li the values are temporary, until defined in CUSS; see Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
-//                 
-//  @date 27.01.2016
-//  @li V1.3.3
-//  @li Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
-//  @li changed values for DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
-//                 
-//  @date 27.01.2016
-//  @li V1.3.4
-//  @li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
-//  @li added BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE
-//                 
-//----------------------------------------------------------------------------
+/**
+File:      codes.idl
+
+Purpose:   Definition of CUSS codes
+
+Date:      17.06.2013
+
+Version:   1.3
+
+Author:    IATA Passenger Experience Management Group: CUWG CUSS-TSG
+
+Copyright(c) 2003,2009,2013 International Air Transport Association, All Rights Reserved
+
+Note:      Please refer to the CUSS 1.3 Technical Specification for more information
+
+Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
+
+@date 24.01.2013
+@li V1.3
+@li Mantis 0025566: WeightScale: support standalone Scale (BaggageScale CUSS1.3)
+@li added CUSS1.3 dataStatus and statusCodes
+
+@date 26.11.2013
+@li V1.3.1
+@li Mantis 0029957: ConveyorComp: CUSS1.3 Conveyor (without ConveyorAEA)
+@li use full CUSS1.3 IDL + Materna extension statuscodes: BAGGAGE_INTRUSION, BAGGAGE_INTRUSION_FRONT, BAGGAGE_INTRUSION_BACK, 
+
+@date 26.01.2016
+@li V1.3.2
+@li Mantis 0037350: integration of EZ-Access device for KIS kiosk
+@li added DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
+@li the values are temporary, until defined in CUSS; see Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
+               
+@date 27.01.2016
+@li V1.3.3
+@li Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
+@li changed values for DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
+               
+@date 17.02.2016
+@li V1.3.4
+@li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
+@li added BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE
+               
+@date 18.02.2016
+@li V1.3.5
+@li Mantis 0037187: CUSS-Spec: DS_TYPES_SSML11= 25001
+@li added DS_TYPES_SSML10; DS_TYPES_SSML11
+- clarified: use DS_TYPES_SSML11= 25010 instead of DS_TYPES_SSML11= 25001
+               
+*/
 
 #ifndef CODES_IDL
 #define CODES_IDL
@@ -155,6 +160,9 @@ module datastatus
     const long DS_TYPES_EPAYMENT                 = 23000; /**< E-Payment: CuPaySchema.xsd */
     const long DS_TYPES_ILLUMINATION             = 24000; /**< LED CUSS_ILLUMINATION.xsd */
 
+    const long DS_TYPES_SSML10                   = 25000; /**< SSML Specification supported by the Announcement Interface */
+    const long DS_TYPES_SSML11                   = 25010; /**< SSML Specification supported by the Announcement Interface */
+
     const long DS_TYPES_KEY                      = 26000; /**< key pressed */
     const long DS_TYPES_KEY_DOWN                 = 26010; /**< key initially pressed */
     const long DS_TYPES_KEY_UP                   = 26020; /**< key released */
-- 
2.41.0.windows.1

